[2023-12-30T09:09:10.420+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dbt_model.run_dbt_bronze_models scheduled__2022-09-10T00:00:00+00:00 [queued]>
[2023-12-30T09:09:10.428+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dbt_model.run_dbt_bronze_models scheduled__2022-09-10T00:00:00+00:00 [queued]>
[2023-12-30T09:09:10.428+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-12-30T09:09:10.428+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 6
[2023-12-30T09:09:10.428+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-12-30T09:09:10.438+0000] {taskinstance.py:1383} INFO - Executing <Task(BashOperator): run_dbt_bronze_models> on 2022-09-10 00:00:00+00:00
[2023-12-30T09:09:10.441+0000] {standard_task_runner.py:55} INFO - Started process 293 to run task
[2023-12-30T09:09:10.443+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'dbt_model', 'run_dbt_bronze_models', 'scheduled__2022-09-10T00:00:00+00:00', '--job-id', '105', '--raw', '--subdir', 'DAGS_FOLDER/run_dbt._models.py', '--cfg-path', '/tmp/tmpxdkoe23u']
[2023-12-30T09:09:10.443+0000] {standard_task_runner.py:83} INFO - Job 105: Subtask run_dbt_bronze_models
[2023-12-30T09:09:10.483+0000] {task_command.py:376} INFO - Running <TaskInstance: dbt_model.run_dbt_bronze_models scheduled__2022-09-10T00:00:00+00:00 [running]> on host 683423f116e7
[2023-12-30T09:09:10.530+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=alonso_md
AIRFLOW_CTX_DAG_ID=dbt_model
AIRFLOW_CTX_TASK_ID=run_dbt_bronze_models
AIRFLOW_CTX_EXECUTION_DATE=2022-09-10T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-09-10T00:00:00+00:00
[2023-12-30T09:09:10.533+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-12-30T09:09:10.534+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', "    dbt run     --profiles-dir //opt/***/dags/../dbt/     --project-dir ///opt/***/dags/../dbt//dwh     --target prod_*** --vars '{ target_date: 2022-09-10 }' --select bronze"]
[2023-12-30T09:09:10.541+0000] {subprocess.py:86} INFO - Output:
[2023-12-30T09:09:12.619+0000] {subprocess.py:93} INFO - [0m09:09:12  Running with dbt=1.4.6
[2023-12-30T09:09:12.668+0000] {subprocess.py:93} INFO - [0m09:09:12  Unable to do partial parsing because saved manifest not found. Starting full parse.
[2023-12-30T09:09:14.192+0000] {subprocess.py:93} INFO - [0m09:09:14  Found 8 models, 15 tests, 0 snapshots, 0 analyses, 703 macros, 0 operations, 0 seed files, 3 sources, 0 exposures, 0 metrics
[2023-12-30T09:09:14.194+0000] {subprocess.py:93} INFO - [0m09:09:14
[2023-12-30T09:09:14.312+0000] {subprocess.py:93} INFO - [0m09:09:14  Concurrency: 4 threads (target='prod_***')
[2023-12-30T09:09:14.312+0000] {subprocess.py:93} INFO - [0m09:09:14
[2023-12-30T09:09:14.322+0000] {subprocess.py:93} INFO - [0m09:09:14  1 of 3 START sql view model bronze.raw_daily_customer_cdc ...................... [RUN]
[2023-12-30T09:09:14.323+0000] {subprocess.py:93} INFO - [0m09:09:14  2 of 3 START sql view model bronze.raw_daily_customer_drivers .................. [RUN]
[2023-12-30T09:09:14.325+0000] {subprocess.py:93} INFO - [0m09:09:14  3 of 3 START sql view model bronze.raw_daily_loan_transactions ................. [RUN]
[2023-12-30T09:09:14.492+0000] {subprocess.py:93} INFO - [0m09:09:14  3 of 3 ERROR creating sql view model bronze.raw_daily_loan_transactions ........ [[31mERROR[0m in 0.16s]
[2023-12-30T09:09:14.495+0000] {subprocess.py:93} INFO - [0m09:09:14  2 of 3 ERROR creating sql view model bronze.raw_daily_customer_drivers ......... [[31mERROR[0m in 0.17s]
[2023-12-30T09:09:14.496+0000] {subprocess.py:93} INFO - [0m09:09:14  1 of 3 ERROR creating sql view model bronze.raw_daily_customer_cdc ............. [[31mERROR[0m in 0.17s]
[2023-12-30T09:09:14.570+0000] {subprocess.py:93} INFO - [0m09:09:14
[2023-12-30T09:09:14.571+0000] {subprocess.py:93} INFO - [0m09:09:14  Finished running 3 view models in 0 hours 0 minutes and 0.38 seconds (0.38s).
[2023-12-30T09:09:14.585+0000] {subprocess.py:93} INFO - [0m09:09:14
[2023-12-30T09:09:14.586+0000] {subprocess.py:93} INFO - [0m09:09:14  [31mCompleted with 3 errors and 0 warnings:[0m
[2023-12-30T09:09:14.586+0000] {subprocess.py:93} INFO - [0m09:09:14
[2023-12-30T09:09:14.587+0000] {subprocess.py:93} INFO - [0m09:09:14  [33mDatabase Error in model raw_daily_loan_transactions (models/bronze/raw_daily_loan_transactions.sql)[0m
[2023-12-30T09:09:14.587+0000] {subprocess.py:93} INFO - [0m09:09:14    relation "airbyte.daily_loan_transactions" does not exist
[2023-12-30T09:09:14.588+0000] {subprocess.py:93} INFO - [0m09:09:14    LINE 9:     FROM "dwh"."airbyte"."daily_loan_transactions"
[2023-12-30T09:09:14.588+0000] {subprocess.py:93} INFO - [0m09:09:14                     ^
[2023-12-30T09:09:14.588+0000] {subprocess.py:93} INFO - [0m09:09:14    compiled Code at target/run/dwh/models/bronze/raw_daily_loan_transactions.sql
[2023-12-30T09:09:14.589+0000] {subprocess.py:93} INFO - [0m09:09:14
[2023-12-30T09:09:14.589+0000] {subprocess.py:93} INFO - [0m09:09:14  [33mDatabase Error in model raw_daily_customer_drivers (models/bronze/raw_daily_customer_drivers.sql)[0m
[2023-12-30T09:09:14.590+0000] {subprocess.py:93} INFO - [0m09:09:14    relation "airbyte.daily_customer_drivers" does not exist
[2023-12-30T09:09:14.590+0000] {subprocess.py:93} INFO - [0m09:09:14    LINE 9:     FROM "dwh"."airbyte"."daily_customer_drivers"
[2023-12-30T09:09:14.590+0000] {subprocess.py:93} INFO - [0m09:09:14                     ^
[2023-12-30T09:09:14.591+0000] {subprocess.py:93} INFO - [0m09:09:14    compiled Code at target/run/dwh/models/bronze/raw_daily_customer_drivers.sql
[2023-12-30T09:09:14.591+0000] {subprocess.py:93} INFO - [0m09:09:14
[2023-12-30T09:09:14.591+0000] {subprocess.py:93} INFO - [0m09:09:14  [33mDatabase Error in model raw_daily_customer_cdc (models/bronze/raw_daily_customer_cdc.sql)[0m
[2023-12-30T09:09:14.592+0000] {subprocess.py:93} INFO - [0m09:09:14    relation "airbyte.daily_customer_information_cdc" does not exist
[2023-12-30T09:09:14.592+0000] {subprocess.py:93} INFO - [0m09:09:14    LINE 9:     FROM "dwh"."airbyte"."daily_customer_information_cdc"
[2023-12-30T09:09:14.592+0000] {subprocess.py:93} INFO - [0m09:09:14                     ^
[2023-12-30T09:09:14.592+0000] {subprocess.py:93} INFO - [0m09:09:14    compiled Code at target/run/dwh/models/bronze/raw_daily_customer_cdc.sql
[2023-12-30T09:09:14.593+0000] {subprocess.py:93} INFO - [0m09:09:14
[2023-12-30T09:09:14.593+0000] {subprocess.py:93} INFO - [0m09:09:14  Done. PASS=0 WARN=0 ERROR=3 SKIP=0 TOTAL=3
[2023-12-30T09:09:16.090+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2023-12-30T09:09:16.101+0000] {taskinstance.py:1851} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 197, in execute
    f'Bash command failed. The command returned a non-zero exit code {result.exit_code}.'
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2023-12-30T09:09:16.107+0000] {taskinstance.py:1406} INFO - Marking task as UP_FOR_RETRY. dag_id=dbt_model, task_id=run_dbt_bronze_models, execution_date=20220910T000000, start_date=20231230T090910, end_date=20231230T090916
[2023-12-30T09:09:16.128+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 105 for task run_dbt_bronze_models (Bash command failed. The command returned a non-zero exit code 1.; 293)
[2023-12-30T09:09:16.138+0000] {local_task_job.py:164} INFO - Task exited with return code 1
[2023-12-30T09:09:16.169+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-12-30T09:10:55.077+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dbt_model.run_dbt_bronze_models scheduled__2022-09-10T00:00:00+00:00 [queued]>
[2023-12-30T09:10:55.086+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dbt_model.run_dbt_bronze_models scheduled__2022-09-10T00:00:00+00:00 [queued]>
[2023-12-30T09:10:55.086+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-12-30T09:10:55.086+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 6
[2023-12-30T09:10:55.086+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-12-30T09:10:55.097+0000] {taskinstance.py:1383} INFO - Executing <Task(BashOperator): run_dbt_bronze_models> on 2022-09-10 00:00:00+00:00
[2023-12-30T09:10:55.100+0000] {standard_task_runner.py:55} INFO - Started process 445 to run task
[2023-12-30T09:10:55.103+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'dbt_model', 'run_dbt_bronze_models', 'scheduled__2022-09-10T00:00:00+00:00', '--job-id', '109', '--raw', '--subdir', 'DAGS_FOLDER/run_dbt._models.py', '--cfg-path', '/tmp/tmp53s10rws']
[2023-12-30T09:10:55.103+0000] {standard_task_runner.py:83} INFO - Job 109: Subtask run_dbt_bronze_models
[2023-12-30T09:10:55.144+0000] {task_command.py:376} INFO - Running <TaskInstance: dbt_model.run_dbt_bronze_models scheduled__2022-09-10T00:00:00+00:00 [running]> on host 683423f116e7
[2023-12-30T09:10:55.199+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=alonso_md
AIRFLOW_CTX_DAG_ID=dbt_model
AIRFLOW_CTX_TASK_ID=run_dbt_bronze_models
AIRFLOW_CTX_EXECUTION_DATE=2022-09-10T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-09-10T00:00:00+00:00
[2023-12-30T09:10:55.200+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-12-30T09:10:55.201+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', "    dbt run     --profiles-dir //opt/***/dags/../dbt/     --project-dir ///opt/***/dags/../dbt//dwh     --target prod_*** --vars '{ target_date: 2022-09-10 }' --select bronze"]
[2023-12-30T09:10:55.209+0000] {subprocess.py:86} INFO - Output:
[2023-12-30T09:10:57.083+0000] {subprocess.py:93} INFO - [0m09:10:57  Running with dbt=1.4.6
[2023-12-30T09:10:57.302+0000] {subprocess.py:93} INFO - [0m09:10:57  Found 8 models, 15 tests, 0 snapshots, 0 analyses, 703 macros, 0 operations, 0 seed files, 3 sources, 0 exposures, 0 metrics
[2023-12-30T09:10:57.304+0000] {subprocess.py:93} INFO - [0m09:10:57
[2023-12-30T09:10:57.425+0000] {subprocess.py:93} INFO - [0m09:10:57  Concurrency: 4 threads (target='prod_***')
[2023-12-30T09:10:57.425+0000] {subprocess.py:93} INFO - [0m09:10:57
[2023-12-30T09:10:57.436+0000] {subprocess.py:93} INFO - [0m09:10:57  1 of 3 START sql view model bronze.raw_daily_customer_cdc ...................... [RUN]
[2023-12-30T09:10:57.437+0000] {subprocess.py:93} INFO - [0m09:10:57  2 of 3 START sql view model bronze.raw_daily_customer_drivers .................. [RUN]
[2023-12-30T09:10:57.439+0000] {subprocess.py:93} INFO - [0m09:10:57  3 of 3 START sql view model bronze.raw_daily_loan_transactions ................. [RUN]
[2023-12-30T09:10:57.544+0000] {subprocess.py:93} INFO - [0m09:10:57  3 of 3 ERROR creating sql view model bronze.raw_daily_loan_transactions ........ [[31mERROR[0m in 0.09s]
[2023-12-30T09:10:57.547+0000] {subprocess.py:93} INFO - [0m09:10:57  1 of 3 ERROR creating sql view model bronze.raw_daily_customer_cdc ............. [[31mERROR[0m in 0.11s]
[2023-12-30T09:10:57.548+0000] {subprocess.py:93} INFO - [0m09:10:57  2 of 3 ERROR creating sql view model bronze.raw_daily_customer_drivers ......... [[31mERROR[0m in 0.11s]
[2023-12-30T09:10:57.641+0000] {subprocess.py:93} INFO - [0m09:10:57
[2023-12-30T09:10:57.642+0000] {subprocess.py:93} INFO - [0m09:10:57  Finished running 3 view models in 0 hours 0 minutes and 0.34 seconds (0.34s).
[2023-12-30T09:10:57.654+0000] {subprocess.py:93} INFO - [0m09:10:57
[2023-12-30T09:10:57.654+0000] {subprocess.py:93} INFO - [0m09:10:57  [31mCompleted with 3 errors and 0 warnings:[0m
[2023-12-30T09:10:57.654+0000] {subprocess.py:93} INFO - [0m09:10:57
[2023-12-30T09:10:57.655+0000] {subprocess.py:93} INFO - [0m09:10:57  [33mDatabase Error in model raw_daily_loan_transactions (models/bronze/raw_daily_loan_transactions.sql)[0m
[2023-12-30T09:10:57.655+0000] {subprocess.py:93} INFO - [0m09:10:57    relation "airbyte.daily_loan_transactions" does not exist
[2023-12-30T09:10:57.655+0000] {subprocess.py:93} INFO - [0m09:10:57    LINE 9:     FROM "dwh"."airbyte"."daily_loan_transactions"
[2023-12-30T09:10:57.656+0000] {subprocess.py:93} INFO - [0m09:10:57                     ^
[2023-12-30T09:10:57.656+0000] {subprocess.py:93} INFO - [0m09:10:57    compiled Code at target/run/dwh/models/bronze/raw_daily_loan_transactions.sql
[2023-12-30T09:10:57.656+0000] {subprocess.py:93} INFO - [0m09:10:57
[2023-12-30T09:10:57.657+0000] {subprocess.py:93} INFO - [0m09:10:57  [33mDatabase Error in model raw_daily_customer_cdc (models/bronze/raw_daily_customer_cdc.sql)[0m
[2023-12-30T09:10:57.657+0000] {subprocess.py:93} INFO - [0m09:10:57    relation "airbyte.daily_customer_information_cdc" does not exist
[2023-12-30T09:10:57.657+0000] {subprocess.py:93} INFO - [0m09:10:57    LINE 9:     FROM "dwh"."airbyte"."daily_customer_information_cdc"
[2023-12-30T09:10:57.657+0000] {subprocess.py:93} INFO - [0m09:10:57                     ^
[2023-12-30T09:10:57.658+0000] {subprocess.py:93} INFO - [0m09:10:57    compiled Code at target/run/dwh/models/bronze/raw_daily_customer_cdc.sql
[2023-12-30T09:10:57.658+0000] {subprocess.py:93} INFO - [0m09:10:57
[2023-12-30T09:10:57.658+0000] {subprocess.py:93} INFO - [0m09:10:57  [33mDatabase Error in model raw_daily_customer_drivers (models/bronze/raw_daily_customer_drivers.sql)[0m
[2023-12-30T09:10:57.659+0000] {subprocess.py:93} INFO - [0m09:10:57    relation "airbyte.daily_customer_drivers" does not exist
[2023-12-30T09:10:57.659+0000] {subprocess.py:93} INFO - [0m09:10:57    LINE 9:     FROM "dwh"."airbyte"."daily_customer_drivers"
[2023-12-30T09:10:57.659+0000] {subprocess.py:93} INFO - [0m09:10:57                     ^
[2023-12-30T09:10:57.659+0000] {subprocess.py:93} INFO - [0m09:10:57    compiled Code at target/run/dwh/models/bronze/raw_daily_customer_drivers.sql
[2023-12-30T09:10:57.659+0000] {subprocess.py:93} INFO - [0m09:10:57
[2023-12-30T09:10:57.660+0000] {subprocess.py:93} INFO - [0m09:10:57  Done. PASS=0 WARN=0 ERROR=3 SKIP=0 TOTAL=3
[2023-12-30T09:10:59.003+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2023-12-30T09:10:59.011+0000] {taskinstance.py:1851} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 197, in execute
    f'Bash command failed. The command returned a non-zero exit code {result.exit_code}.'
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2023-12-30T09:10:59.014+0000] {taskinstance.py:1406} INFO - Marking task as UP_FOR_RETRY. dag_id=dbt_model, task_id=run_dbt_bronze_models, execution_date=20220910T000000, start_date=20231230T091055, end_date=20231230T091059
[2023-12-30T09:10:59.027+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 109 for task run_dbt_bronze_models (Bash command failed. The command returned a non-zero exit code 1.; 445)
[2023-12-30T09:10:59.050+0000] {local_task_job.py:164} INFO - Task exited with return code 1
[2023-12-30T09:10:59.078+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-12-30T09:46:18.749+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dbt_model.run_dbt_bronze_models scheduled__2022-09-10T00:00:00+00:00 [queued]>
[2023-12-30T09:46:18.764+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dbt_model.run_dbt_bronze_models scheduled__2022-09-10T00:00:00+00:00 [queued]>
[2023-12-30T09:46:18.764+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-12-30T09:46:18.764+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 6
[2023-12-30T09:46:18.764+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-12-30T09:46:18.777+0000] {taskinstance.py:1383} INFO - Executing <Task(BashOperator): run_dbt_bronze_models> on 2022-09-10 00:00:00+00:00
[2023-12-30T09:46:18.781+0000] {standard_task_runner.py:55} INFO - Started process 2241 to run task
[2023-12-30T09:46:18.784+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'dbt_model', 'run_dbt_bronze_models', 'scheduled__2022-09-10T00:00:00+00:00', '--job-id', '113', '--raw', '--subdir', 'DAGS_FOLDER/run_dbt._models.py', '--cfg-path', '/tmp/tmpxwbkzv1r']
[2023-12-30T09:46:18.785+0000] {standard_task_runner.py:83} INFO - Job 113: Subtask run_dbt_bronze_models
[2023-12-30T09:46:18.831+0000] {task_command.py:376} INFO - Running <TaskInstance: dbt_model.run_dbt_bronze_models scheduled__2022-09-10T00:00:00+00:00 [running]> on host 683423f116e7
[2023-12-30T09:46:18.885+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=alonso_md
AIRFLOW_CTX_DAG_ID=dbt_model
AIRFLOW_CTX_TASK_ID=run_dbt_bronze_models
AIRFLOW_CTX_EXECUTION_DATE=2022-09-10T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-09-10T00:00:00+00:00
[2023-12-30T09:46:18.890+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-12-30T09:46:18.891+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', "    dbt run     --profiles-dir //opt/***/dags/../dbt/     --project-dir ///opt/***/dags/../dbt//dwh     --target prod_*** --vars '{ target_date: 2022-09-10 }' --select bronze"]
[2023-12-30T09:46:18.900+0000] {subprocess.py:86} INFO - Output:
[2023-12-30T09:46:20.942+0000] {subprocess.py:93} INFO - [0m09:46:20  Running with dbt=1.4.6
[2023-12-30T09:46:21.105+0000] {subprocess.py:93} INFO - [0m09:46:21  Found 8 models, 15 tests, 0 snapshots, 0 analyses, 703 macros, 0 operations, 0 seed files, 3 sources, 0 exposures, 0 metrics
[2023-12-30T09:46:21.107+0000] {subprocess.py:93} INFO - [0m09:46:21
[2023-12-30T09:46:21.173+0000] {subprocess.py:93} INFO - [0m09:46:21  Concurrency: 4 threads (target='prod_***')
[2023-12-30T09:46:21.173+0000] {subprocess.py:93} INFO - [0m09:46:21
[2023-12-30T09:46:21.180+0000] {subprocess.py:93} INFO - [0m09:46:21  1 of 3 START sql view model bronze.raw_daily_customer_cdc ...................... [RUN]
[2023-12-30T09:46:21.180+0000] {subprocess.py:93} INFO - [0m09:46:21  2 of 3 START sql view model bronze.raw_daily_customer_drivers .................. [RUN]
[2023-12-30T09:46:21.182+0000] {subprocess.py:93} INFO - [0m09:46:21  3 of 3 START sql view model bronze.raw_daily_loan_transactions ................. [RUN]
[2023-12-30T09:46:21.261+0000] {subprocess.py:93} INFO - [0m09:46:21  1 of 3 ERROR creating sql view model bronze.raw_daily_customer_cdc ............. [[31mERROR[0m in 0.08s]
[2023-12-30T09:46:21.262+0000] {subprocess.py:93} INFO - [0m09:46:21  3 of 3 ERROR creating sql view model bronze.raw_daily_loan_transactions ........ [[31mERROR[0m in 0.08s]
[2023-12-30T09:46:21.264+0000] {subprocess.py:93} INFO - [0m09:46:21  2 of 3 ERROR creating sql view model bronze.raw_daily_customer_drivers ......... [[31mERROR[0m in 0.08s]
[2023-12-30T09:46:21.288+0000] {subprocess.py:93} INFO - [0m09:46:21
[2023-12-30T09:46:21.288+0000] {subprocess.py:93} INFO - [0m09:46:21  Finished running 3 view models in 0 hours 0 minutes and 0.18 seconds (0.18s).
[2023-12-30T09:46:21.298+0000] {subprocess.py:93} INFO - [0m09:46:21
[2023-12-30T09:46:21.299+0000] {subprocess.py:93} INFO - [0m09:46:21  [31mCompleted with 3 errors and 0 warnings:[0m
[2023-12-30T09:46:21.299+0000] {subprocess.py:93} INFO - [0m09:46:21
[2023-12-30T09:46:21.300+0000] {subprocess.py:93} INFO - [0m09:46:21  [33mDatabase Error in model raw_daily_customer_cdc (models/bronze/raw_daily_customer_cdc.sql)[0m
[2023-12-30T09:46:21.300+0000] {subprocess.py:93} INFO - [0m09:46:21    column "date" does not exist
[2023-12-30T09:46:21.300+0000] {subprocess.py:93} INFO - [0m09:46:21    LINE 12:     cast(date as date) as date,
[2023-12-30T09:46:21.300+0000] {subprocess.py:93} INFO - [0m09:46:21                      ^
[2023-12-30T09:46:21.301+0000] {subprocess.py:93} INFO - [0m09:46:21    HINT:  Perhaps you meant to reference the column "raw_daily_customer_cdc.﻿date".
[2023-12-30T09:46:21.301+0000] {subprocess.py:93} INFO - [0m09:46:21    compiled Code at target/run/dwh/models/bronze/raw_daily_customer_cdc.sql
[2023-12-30T09:46:21.301+0000] {subprocess.py:93} INFO - [0m09:46:21
[2023-12-30T09:46:21.301+0000] {subprocess.py:93} INFO - [0m09:46:21  [33mDatabase Error in model raw_daily_loan_transactions (models/bronze/raw_daily_loan_transactions.sql)[0m
[2023-12-30T09:46:21.302+0000] {subprocess.py:93} INFO - [0m09:46:21    column "date" does not exist
[2023-12-30T09:46:21.302+0000] {subprocess.py:93} INFO - [0m09:46:21    LINE 12:     cast(date as date) as date,
[2023-12-30T09:46:21.302+0000] {subprocess.py:93} INFO - [0m09:46:21                      ^
[2023-12-30T09:46:21.302+0000] {subprocess.py:93} INFO - [0m09:46:21    HINT:  Perhaps you meant to reference the column "raw_daily_loan_transactions.﻿date".
[2023-12-30T09:46:21.303+0000] {subprocess.py:93} INFO - [0m09:46:21    compiled Code at target/run/dwh/models/bronze/raw_daily_loan_transactions.sql
[2023-12-30T09:46:21.303+0000] {subprocess.py:93} INFO - [0m09:46:21
[2023-12-30T09:46:21.303+0000] {subprocess.py:93} INFO - [0m09:46:21  [33mDatabase Error in model raw_daily_customer_drivers (models/bronze/raw_daily_customer_drivers.sql)[0m
[2023-12-30T09:46:21.303+0000] {subprocess.py:93} INFO - [0m09:46:21    column "date" does not exist
[2023-12-30T09:46:21.304+0000] {subprocess.py:93} INFO - [0m09:46:21    LINE 12:     cast(date as date) as date,
[2023-12-30T09:46:21.304+0000] {subprocess.py:93} INFO - [0m09:46:21                      ^
[2023-12-30T09:46:21.304+0000] {subprocess.py:93} INFO - [0m09:46:21    HINT:  Perhaps you meant to reference the column "raw_daily_customer_drivers.﻿date".
[2023-12-30T09:46:21.304+0000] {subprocess.py:93} INFO - [0m09:46:21    compiled Code at target/run/dwh/models/bronze/raw_daily_customer_drivers.sql
[2023-12-30T09:46:21.305+0000] {subprocess.py:93} INFO - [0m09:46:21
[2023-12-30T09:46:21.305+0000] {subprocess.py:93} INFO - [0m09:46:21  Done. PASS=0 WARN=0 ERROR=3 SKIP=0 TOTAL=3
[2023-12-30T09:46:22.779+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2023-12-30T09:46:22.788+0000] {taskinstance.py:1851} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 197, in execute
    f'Bash command failed. The command returned a non-zero exit code {result.exit_code}.'
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2023-12-30T09:46:22.791+0000] {taskinstance.py:1406} INFO - Marking task as UP_FOR_RETRY. dag_id=dbt_model, task_id=run_dbt_bronze_models, execution_date=20220910T000000, start_date=20231230T094618, end_date=20231230T094622
[2023-12-30T09:46:22.802+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 113 for task run_dbt_bronze_models (Bash command failed. The command returned a non-zero exit code 1.; 2241)
[2023-12-30T09:46:22.811+0000] {local_task_job.py:164} INFO - Task exited with return code 1
[2023-12-30T09:46:22.840+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-12-30T10:25:55.366+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dbt_model.run_dbt_bronze_models scheduled__2022-09-10T00:00:00+00:00 [queued]>
[2023-12-30T10:25:55.390+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dbt_model.run_dbt_bronze_models scheduled__2022-09-10T00:00:00+00:00 [queued]>
[2023-12-30T10:25:55.390+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-12-30T10:25:55.391+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 6
[2023-12-30T10:25:55.391+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-12-30T10:25:55.423+0000] {taskinstance.py:1383} INFO - Executing <Task(BashOperator): run_dbt_bronze_models> on 2022-09-10 00:00:00+00:00
[2023-12-30T10:25:55.436+0000] {standard_task_runner.py:55} INFO - Started process 4250 to run task
[2023-12-30T10:25:55.442+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'dbt_model', 'run_dbt_bronze_models', 'scheduled__2022-09-10T00:00:00+00:00', '--job-id', '117', '--raw', '--subdir', 'DAGS_FOLDER/run_dbt._models.py', '--cfg-path', '/tmp/tmpi6lrmfx2']
[2023-12-30T10:25:55.443+0000] {standard_task_runner.py:83} INFO - Job 117: Subtask run_dbt_bronze_models
[2023-12-30T10:25:55.554+0000] {task_command.py:376} INFO - Running <TaskInstance: dbt_model.run_dbt_bronze_models scheduled__2022-09-10T00:00:00+00:00 [running]> on host 683423f116e7
[2023-12-30T10:25:55.652+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=alonso_md
AIRFLOW_CTX_DAG_ID=dbt_model
AIRFLOW_CTX_TASK_ID=run_dbt_bronze_models
AIRFLOW_CTX_EXECUTION_DATE=2022-09-10T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-09-10T00:00:00+00:00
[2023-12-30T10:25:55.653+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-12-30T10:25:55.653+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', "    dbt run     --profiles-dir //opt/***/dags/../dbt/     --project-dir ///opt/***/dags/../dbt//dwh     --target prod_*** --vars '{ target_date: 2022-09-10 }' --select bronze"]
[2023-12-30T10:25:55.665+0000] {subprocess.py:86} INFO - Output:
[2023-12-30T10:25:57.568+0000] {subprocess.py:93} INFO - [0m10:25:57  Running with dbt=1.4.6
[2023-12-30T10:25:57.788+0000] {subprocess.py:93} INFO - [0m10:25:57  Found 8 models, 15 tests, 0 snapshots, 0 analyses, 703 macros, 0 operations, 0 seed files, 3 sources, 0 exposures, 0 metrics
[2023-12-30T10:25:57.790+0000] {subprocess.py:93} INFO - [0m10:25:57
[2023-12-30T10:25:57.878+0000] {subprocess.py:93} INFO - [0m10:25:57  Concurrency: 4 threads (target='prod_***')
[2023-12-30T10:25:57.879+0000] {subprocess.py:93} INFO - [0m10:25:57
[2023-12-30T10:25:57.893+0000] {subprocess.py:93} INFO - [0m10:25:57  1 of 3 START sql view model bronze.raw_daily_customer_cdc ...................... [RUN]
[2023-12-30T10:25:57.896+0000] {subprocess.py:93} INFO - [0m10:25:57  2 of 3 START sql view model bronze.raw_daily_customer_drivers .................. [RUN]
[2023-12-30T10:25:57.900+0000] {subprocess.py:93} INFO - [0m10:25:57  3 of 3 START sql view model bronze.raw_daily_loan_transactions ................. [RUN]
[2023-12-30T10:25:57.993+0000] {subprocess.py:93} INFO - [0m10:25:57  2 of 3 ERROR creating sql view model bronze.raw_daily_customer_drivers ......... [[31mERROR[0m in 0.09s]
[2023-12-30T10:25:58.006+0000] {subprocess.py:93} INFO - [0m10:25:58  3 of 3 ERROR creating sql view model bronze.raw_daily_loan_transactions ........ [[31mERROR[0m in 0.10s]
[2023-12-30T10:25:58.076+0000] {subprocess.py:93} INFO - [0m10:25:58  1 of 3 OK created sql view model bronze.raw_daily_customer_cdc ................. [[32mCREATE VIEW[0m in 0.18s]
[2023-12-30T10:25:58.100+0000] {subprocess.py:93} INFO - [0m10:25:58
[2023-12-30T10:25:58.100+0000] {subprocess.py:93} INFO - [0m10:25:58  Finished running 3 view models in 0 hours 0 minutes and 0.31 seconds (0.31s).
[2023-12-30T10:25:58.113+0000] {subprocess.py:93} INFO - [0m10:25:58
[2023-12-30T10:25:58.113+0000] {subprocess.py:93} INFO - [0m10:25:58  [31mCompleted with 2 errors and 0 warnings:[0m
[2023-12-30T10:25:58.114+0000] {subprocess.py:93} INFO - [0m10:25:58
[2023-12-30T10:25:58.114+0000] {subprocess.py:93} INFO - [0m10:25:58  [33mDatabase Error in model raw_daily_customer_drivers (models/bronze/raw_daily_customer_drivers.sql)[0m
[2023-12-30T10:25:58.114+0000] {subprocess.py:93} INFO - [0m10:25:58    column "date" does not exist
[2023-12-30T10:25:58.115+0000] {subprocess.py:93} INFO - [0m10:25:58    LINE 12:     cast(date as date) as date,
[2023-12-30T10:25:58.115+0000] {subprocess.py:93} INFO - [0m10:25:58                      ^
[2023-12-30T10:25:58.115+0000] {subprocess.py:93} INFO - [0m10:25:58    HINT:  Perhaps you meant to reference the column "raw_daily_customer_drivers.﻿date".
[2023-12-30T10:25:58.116+0000] {subprocess.py:93} INFO - [0m10:25:58    compiled Code at target/run/dwh/models/bronze/raw_daily_customer_drivers.sql
[2023-12-30T10:25:58.116+0000] {subprocess.py:93} INFO - [0m10:25:58
[2023-12-30T10:25:58.116+0000] {subprocess.py:93} INFO - [0m10:25:58  [33mDatabase Error in model raw_daily_loan_transactions (models/bronze/raw_daily_loan_transactions.sql)[0m
[2023-12-30T10:25:58.117+0000] {subprocess.py:93} INFO - [0m10:25:58    column "date" does not exist
[2023-12-30T10:25:58.117+0000] {subprocess.py:93} INFO - [0m10:25:58    LINE 12:     cast(date as date) as date,
[2023-12-30T10:25:58.118+0000] {subprocess.py:93} INFO - [0m10:25:58                      ^
[2023-12-30T10:25:58.118+0000] {subprocess.py:93} INFO - [0m10:25:58    HINT:  Perhaps you meant to reference the column "raw_daily_loan_transactions.﻿date".
[2023-12-30T10:25:58.118+0000] {subprocess.py:93} INFO - [0m10:25:58    compiled Code at target/run/dwh/models/bronze/raw_daily_loan_transactions.sql
[2023-12-30T10:25:58.119+0000] {subprocess.py:93} INFO - [0m10:25:58
[2023-12-30T10:25:58.119+0000] {subprocess.py:93} INFO - [0m10:25:58  Done. PASS=1 WARN=0 ERROR=2 SKIP=0 TOTAL=3
[2023-12-30T10:25:58.974+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2023-12-30T10:25:58.982+0000] {taskinstance.py:1851} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 197, in execute
    f'Bash command failed. The command returned a non-zero exit code {result.exit_code}.'
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2023-12-30T10:25:58.984+0000] {taskinstance.py:1406} INFO - Marking task as UP_FOR_RETRY. dag_id=dbt_model, task_id=run_dbt_bronze_models, execution_date=20220910T000000, start_date=20231230T102555, end_date=20231230T102558
[2023-12-30T10:25:58.995+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 117 for task run_dbt_bronze_models (Bash command failed. The command returned a non-zero exit code 1.; 4250)
[2023-12-30T10:25:59.028+0000] {local_task_job.py:164} INFO - Task exited with return code 1
[2023-12-30T10:25:59.065+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-12-30T10:40:07.414+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dbt_model.run_dbt_bronze_models scheduled__2022-09-10T00:00:00+00:00 [queued]>
[2023-12-30T10:40:07.424+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dbt_model.run_dbt_bronze_models scheduled__2022-09-10T00:00:00+00:00 [queued]>
[2023-12-30T10:40:07.424+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-12-30T10:40:07.424+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 6
[2023-12-30T10:40:07.424+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-12-30T10:40:07.435+0000] {taskinstance.py:1383} INFO - Executing <Task(BashOperator): run_dbt_bronze_models> on 2022-09-10 00:00:00+00:00
[2023-12-30T10:40:07.439+0000] {standard_task_runner.py:55} INFO - Started process 5025 to run task
[2023-12-30T10:40:07.442+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'dbt_model', 'run_dbt_bronze_models', 'scheduled__2022-09-10T00:00:00+00:00', '--job-id', '119', '--raw', '--subdir', 'DAGS_FOLDER/run_dbt._models.py', '--cfg-path', '/tmp/tmp2zhibutk']
[2023-12-30T10:40:07.442+0000] {standard_task_runner.py:83} INFO - Job 119: Subtask run_dbt_bronze_models
[2023-12-30T10:40:07.494+0000] {task_command.py:376} INFO - Running <TaskInstance: dbt_model.run_dbt_bronze_models scheduled__2022-09-10T00:00:00+00:00 [running]> on host 683423f116e7
[2023-12-30T10:40:07.568+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=alonso_md
AIRFLOW_CTX_DAG_ID=dbt_model
AIRFLOW_CTX_TASK_ID=run_dbt_bronze_models
AIRFLOW_CTX_EXECUTION_DATE=2022-09-10T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-09-10T00:00:00+00:00
[2023-12-30T10:40:07.569+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-12-30T10:40:07.570+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', "    dbt run     --profiles-dir //opt/***/dags/../dbt/     --project-dir ///opt/***/dags/../dbt//dwh     --target prod_*** --vars '{ target_date: 2022-09-10 }' --select bronze"]
[2023-12-30T10:40:07.596+0000] {subprocess.py:86} INFO - Output:
[2023-12-30T10:40:09.644+0000] {subprocess.py:93} INFO - [0m10:40:09  Running with dbt=1.4.6
[2023-12-30T10:40:09.785+0000] {subprocess.py:93} INFO - [0m10:40:09  Found 8 models, 15 tests, 0 snapshots, 0 analyses, 703 macros, 0 operations, 0 seed files, 3 sources, 0 exposures, 0 metrics
[2023-12-30T10:40:09.788+0000] {subprocess.py:93} INFO - [0m10:40:09
[2023-12-30T10:40:09.857+0000] {subprocess.py:93} INFO - [0m10:40:09  Concurrency: 4 threads (target='prod_***')
[2023-12-30T10:40:09.857+0000] {subprocess.py:93} INFO - [0m10:40:09
[2023-12-30T10:40:09.862+0000] {subprocess.py:93} INFO - [0m10:40:09  1 of 3 START sql view model bronze.raw_daily_customer_cdc ...................... [RUN]
[2023-12-30T10:40:09.863+0000] {subprocess.py:93} INFO - [0m10:40:09  2 of 3 START sql view model bronze.raw_daily_customer_drivers .................. [RUN]
[2023-12-30T10:40:09.864+0000] {subprocess.py:93} INFO - [0m10:40:09  3 of 3 START sql view model bronze.raw_daily_loan_transactions ................. [RUN]
[2023-12-30T10:40:10.026+0000] {subprocess.py:93} INFO - [0m10:40:10  2 of 3 OK created sql view model bronze.raw_daily_customer_drivers ............. [[32mCREATE VIEW[0m in 0.16s]
[2023-12-30T10:40:10.031+0000] {subprocess.py:93} INFO - [0m10:40:10  3 of 3 OK created sql view model bronze.raw_daily_loan_transactions ............ [[32mCREATE VIEW[0m in 0.16s]
[2023-12-30T10:40:10.039+0000] {subprocess.py:93} INFO - [0m10:40:10  1 of 3 OK created sql view model bronze.raw_daily_customer_cdc ................. [[32mCREATE VIEW[0m in 0.17s]
[2023-12-30T10:40:10.095+0000] {subprocess.py:93} INFO - [0m10:40:10
[2023-12-30T10:40:10.096+0000] {subprocess.py:93} INFO - [0m10:40:10  Finished running 3 view models in 0 hours 0 minutes and 0.31 seconds (0.31s).
[2023-12-30T10:40:10.109+0000] {subprocess.py:93} INFO - [0m10:40:10
[2023-12-30T10:40:10.109+0000] {subprocess.py:93} INFO - [0m10:40:10  [32mCompleted successfully[0m
[2023-12-30T10:40:10.109+0000] {subprocess.py:93} INFO - [0m10:40:10
[2023-12-30T10:40:10.110+0000] {subprocess.py:93} INFO - [0m10:40:10  Done. PASS=3 WARN=0 ERROR=0 SKIP=0 TOTAL=3
[2023-12-30T10:40:11.265+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-12-30T10:40:11.293+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=dbt_model, task_id=run_dbt_bronze_models, execution_date=20220910T000000, start_date=20231230T104007, end_date=20231230T104011
[2023-12-30T10:40:11.311+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-12-30T10:40:11.340+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-12-30T11:56:50.613+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dbt_model.run_dbt_bronze_models scheduled__2022-09-10T00:00:00+00:00 [queued]>
[2023-12-30T11:56:50.628+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dbt_model.run_dbt_bronze_models scheduled__2022-09-10T00:00:00+00:00 [queued]>
[2023-12-30T11:56:50.628+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-12-30T11:56:50.628+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 6
[2023-12-30T11:56:50.628+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-12-30T11:56:50.642+0000] {taskinstance.py:1383} INFO - Executing <Task(BashOperator): run_dbt_bronze_models> on 2022-09-10 00:00:00+00:00
[2023-12-30T11:56:50.647+0000] {standard_task_runner.py:55} INFO - Started process 8979 to run task
[2023-12-30T11:56:50.652+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'dbt_model', 'run_dbt_bronze_models', 'scheduled__2022-09-10T00:00:00+00:00', '--job-id', '127', '--raw', '--subdir', 'DAGS_FOLDER/run_dbt._models.py', '--cfg-path', '/tmp/tmphwukhlqj']
[2023-12-30T11:56:50.653+0000] {standard_task_runner.py:83} INFO - Job 127: Subtask run_dbt_bronze_models
[2023-12-30T11:56:50.710+0000] {task_command.py:376} INFO - Running <TaskInstance: dbt_model.run_dbt_bronze_models scheduled__2022-09-10T00:00:00+00:00 [running]> on host 683423f116e7
[2023-12-30T11:56:50.777+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=alonso_md
AIRFLOW_CTX_DAG_ID=dbt_model
AIRFLOW_CTX_TASK_ID=run_dbt_bronze_models
AIRFLOW_CTX_EXECUTION_DATE=2022-09-10T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-09-10T00:00:00+00:00
[2023-12-30T11:56:50.784+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-12-30T11:56:50.785+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', "    dbt run     --profiles-dir //opt/***/dags/../dbt/     --project-dir ///opt/***/dags/../dbt//dwh     --target prod_*** --vars '{ target_date: 2022-09-10 }' --select bronze"]
[2023-12-30T11:56:50.795+0000] {subprocess.py:86} INFO - Output:
[2023-12-30T11:56:53.023+0000] {subprocess.py:93} INFO - [0m11:56:53  Running with dbt=1.4.6
[2023-12-30T11:56:53.085+0000] {subprocess.py:93} INFO - [0m11:56:53  Unable to do partial parsing because config vars, config profile, or config target have changed
[2023-12-30T11:56:54.774+0000] {subprocess.py:93} INFO - [0m11:56:54  Found 8 models, 15 tests, 0 snapshots, 0 analyses, 703 macros, 0 operations, 0 seed files, 3 sources, 0 exposures, 0 metrics
[2023-12-30T11:56:54.776+0000] {subprocess.py:93} INFO - [0m11:56:54
[2023-12-30T11:56:54.893+0000] {subprocess.py:93} INFO - [0m11:56:54  Concurrency: 4 threads (target='prod_***')
[2023-12-30T11:56:54.893+0000] {subprocess.py:93} INFO - [0m11:56:54
[2023-12-30T11:56:54.902+0000] {subprocess.py:93} INFO - [0m11:56:54  1 of 3 START sql view model bronze.raw_daily_customer_cdc ...................... [RUN]
[2023-12-30T11:56:54.909+0000] {subprocess.py:93} INFO - [0m11:56:54  2 of 3 START sql view model bronze.raw_daily_customer_drivers .................. [RUN]
[2023-12-30T11:56:54.910+0000] {subprocess.py:93} INFO - [0m11:56:54  3 of 3 START sql view model bronze.raw_daily_loan_transactions ................. [RUN]
[2023-12-30T11:56:55.053+0000] {subprocess.py:93} INFO - [0m11:56:55  1 of 3 OK created sql view model bronze.raw_daily_customer_cdc ................. [[32mCREATE VIEW[0m in 0.15s]
[2023-12-30T11:56:55.055+0000] {subprocess.py:93} INFO - [0m11:56:55  2 of 3 OK created sql view model bronze.raw_daily_customer_drivers ............. [[32mCREATE VIEW[0m in 0.14s]
[2023-12-30T11:56:55.057+0000] {subprocess.py:93} INFO - [0m11:56:55  3 of 3 OK created sql view model bronze.raw_daily_loan_transactions ............ [[32mCREATE VIEW[0m in 0.15s]
[2023-12-30T11:56:55.120+0000] {subprocess.py:93} INFO - [0m11:56:55
[2023-12-30T11:56:55.120+0000] {subprocess.py:93} INFO - [0m11:56:55  Finished running 3 view models in 0 hours 0 minutes and 0.34 seconds (0.34s).
[2023-12-30T11:56:55.132+0000] {subprocess.py:93} INFO - [0m11:56:55
[2023-12-30T11:56:55.133+0000] {subprocess.py:93} INFO - [0m11:56:55  [32mCompleted successfully[0m
[2023-12-30T11:56:55.133+0000] {subprocess.py:93} INFO - [0m11:56:55
[2023-12-30T11:56:55.134+0000] {subprocess.py:93} INFO - [0m11:56:55  Done. PASS=3 WARN=0 ERROR=0 SKIP=0 TOTAL=3
[2023-12-30T11:56:56.287+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-12-30T11:56:56.311+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=dbt_model, task_id=run_dbt_bronze_models, execution_date=20220910T000000, start_date=20231230T115650, end_date=20231230T115656
[2023-12-30T11:56:56.335+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-12-30T11:56:56.380+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-12-30T13:47:16.896+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dbt_model.run_dbt_bronze_models scheduled__2022-09-10T00:00:00+00:00 [queued]>
[2023-12-30T13:47:16.905+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dbt_model.run_dbt_bronze_models scheduled__2022-09-10T00:00:00+00:00 [queued]>
[2023-12-30T13:47:16.905+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-12-30T13:47:16.905+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 6
[2023-12-30T13:47:16.905+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-12-30T13:47:16.917+0000] {taskinstance.py:1383} INFO - Executing <Task(BashOperator): run_dbt_bronze_models> on 2022-09-10 00:00:00+00:00
[2023-12-30T13:47:16.921+0000] {standard_task_runner.py:55} INFO - Started process 14514 to run task
[2023-12-30T13:47:16.924+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'dbt_model', 'run_dbt_bronze_models', 'scheduled__2022-09-10T00:00:00+00:00', '--job-id', '136', '--raw', '--subdir', 'DAGS_FOLDER/run_dbt._models.py', '--cfg-path', '/tmp/tmpzf8rjsi5']
[2023-12-30T13:47:16.925+0000] {standard_task_runner.py:83} INFO - Job 136: Subtask run_dbt_bronze_models
[2023-12-30T13:47:16.969+0000] {task_command.py:376} INFO - Running <TaskInstance: dbt_model.run_dbt_bronze_models scheduled__2022-09-10T00:00:00+00:00 [running]> on host 683423f116e7
[2023-12-30T13:47:17.020+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=alonso_md
AIRFLOW_CTX_DAG_ID=dbt_model
AIRFLOW_CTX_TASK_ID=run_dbt_bronze_models
AIRFLOW_CTX_EXECUTION_DATE=2022-09-10T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-09-10T00:00:00+00:00
[2023-12-30T13:47:17.021+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-12-30T13:47:17.021+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', "    dbt run     --profiles-dir //opt/***/dags/../dbt/     --project-dir ///opt/***/dags/../dbt//dwh     --target prod_*** --vars '{ target_date: 2022-09-10 }' --select bronze"]
[2023-12-30T13:47:17.031+0000] {subprocess.py:86} INFO - Output:
[2023-12-30T13:47:19.245+0000] {subprocess.py:93} INFO - [0m13:47:19  Running with dbt=1.4.6
[2023-12-30T13:47:19.304+0000] {subprocess.py:93} INFO - [0m13:47:19  Unable to do partial parsing because config vars, config profile, or config target have changed
[2023-12-30T13:47:20.935+0000] {subprocess.py:93} INFO - [0m13:47:20  Found 8 models, 15 tests, 0 snapshots, 0 analyses, 703 macros, 0 operations, 0 seed files, 3 sources, 0 exposures, 0 metrics
[2023-12-30T13:47:20.938+0000] {subprocess.py:93} INFO - [0m13:47:20
[2023-12-30T13:47:21.061+0000] {subprocess.py:93} INFO - [0m13:47:21  Concurrency: 4 threads (target='prod_***')
[2023-12-30T13:47:21.062+0000] {subprocess.py:93} INFO - [0m13:47:21
[2023-12-30T13:47:21.070+0000] {subprocess.py:93} INFO - [0m13:47:21  1 of 3 START sql view model bronze.raw_daily_customer_cdc ...................... [RUN]
[2023-12-30T13:47:21.071+0000] {subprocess.py:93} INFO - [0m13:47:21  2 of 3 START sql view model bronze.raw_daily_customer_drivers .................. [RUN]
[2023-12-30T13:47:21.074+0000] {subprocess.py:93} INFO - [0m13:47:21  3 of 3 START sql view model bronze.raw_daily_loan_transactions ................. [RUN]
[2023-12-30T13:47:21.246+0000] {subprocess.py:93} INFO - [0m13:47:21  2 of 3 OK created sql view model bronze.raw_daily_customer_drivers ............. [[32mCREATE VIEW[0m in 0.17s]
[2023-12-30T13:47:21.247+0000] {subprocess.py:93} INFO - [0m13:47:21  3 of 3 OK created sql view model bronze.raw_daily_loan_transactions ............ [[32mCREATE VIEW[0m in 0.17s]
[2023-12-30T13:47:21.250+0000] {subprocess.py:93} INFO - [0m13:47:21  1 of 3 OK created sql view model bronze.raw_daily_customer_cdc ................. [[32mCREATE VIEW[0m in 0.18s]
[2023-12-30T13:47:21.279+0000] {subprocess.py:93} INFO - [0m13:47:21
[2023-12-30T13:47:21.279+0000] {subprocess.py:93} INFO - [0m13:47:21  Finished running 3 view models in 0 hours 0 minutes and 0.34 seconds (0.34s).
[2023-12-30T13:47:21.291+0000] {subprocess.py:93} INFO - [0m13:47:21
[2023-12-30T13:47:21.292+0000] {subprocess.py:93} INFO - [0m13:47:21  [32mCompleted successfully[0m
[2023-12-30T13:47:21.292+0000] {subprocess.py:93} INFO - [0m13:47:21
[2023-12-30T13:47:21.292+0000] {subprocess.py:93} INFO - [0m13:47:21  Done. PASS=3 WARN=0 ERROR=0 SKIP=0 TOTAL=3
[2023-12-30T13:47:22.155+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-12-30T13:47:22.178+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=dbt_model, task_id=run_dbt_bronze_models, execution_date=20220910T000000, start_date=20231230T134716, end_date=20231230T134722
[2023-12-30T13:47:22.209+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-12-30T13:47:22.249+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-12-30T14:32:46.400+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dbt_model.run_dbt_bronze_models scheduled__2022-09-10T00:00:00+00:00 [queued]>
[2023-12-30T14:32:46.411+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dbt_model.run_dbt_bronze_models scheduled__2022-09-10T00:00:00+00:00 [queued]>
[2023-12-30T14:32:46.411+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-12-30T14:32:46.411+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 6
[2023-12-30T14:32:46.411+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-12-30T14:32:46.421+0000] {taskinstance.py:1383} INFO - Executing <Task(BashOperator): run_dbt_bronze_models> on 2022-09-10 00:00:00+00:00
[2023-12-30T14:32:46.425+0000] {standard_task_runner.py:55} INFO - Started process 16889 to run task
[2023-12-30T14:32:46.427+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'dbt_model', 'run_dbt_bronze_models', 'scheduled__2022-09-10T00:00:00+00:00', '--job-id', '145', '--raw', '--subdir', 'DAGS_FOLDER/run_dbt._models.py', '--cfg-path', '/tmp/tmph1tadstf']
[2023-12-30T14:32:46.428+0000] {standard_task_runner.py:83} INFO - Job 145: Subtask run_dbt_bronze_models
[2023-12-30T14:32:46.476+0000] {task_command.py:376} INFO - Running <TaskInstance: dbt_model.run_dbt_bronze_models scheduled__2022-09-10T00:00:00+00:00 [running]> on host 683423f116e7
[2023-12-30T14:32:46.530+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=alonso_md
AIRFLOW_CTX_DAG_ID=dbt_model
AIRFLOW_CTX_TASK_ID=run_dbt_bronze_models
AIRFLOW_CTX_EXECUTION_DATE=2022-09-10T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-09-10T00:00:00+00:00
[2023-12-30T14:32:46.535+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-12-30T14:32:46.535+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', "    dbt run     --profiles-dir //opt/***/dags/../dbt/     --project-dir ///opt/***/dags/../dbt//dwh     --target prod_*** --vars '{ target_date: 2022-09-10 }' --select bronze"]
[2023-12-30T14:32:46.544+0000] {subprocess.py:86} INFO - Output:
[2023-12-30T14:32:48.453+0000] {subprocess.py:93} INFO - [0m14:32:48  Running with dbt=1.4.6
[2023-12-30T14:32:48.614+0000] {subprocess.py:93} INFO - [0m14:32:48  Found 8 models, 15 tests, 0 snapshots, 0 analyses, 703 macros, 0 operations, 0 seed files, 3 sources, 0 exposures, 0 metrics
[2023-12-30T14:32:48.616+0000] {subprocess.py:93} INFO - [0m14:32:48
[2023-12-30T14:32:48.699+0000] {subprocess.py:93} INFO - [0m14:32:48  Concurrency: 4 threads (target='prod_***')
[2023-12-30T14:32:48.700+0000] {subprocess.py:93} INFO - [0m14:32:48
[2023-12-30T14:32:48.711+0000] {subprocess.py:93} INFO - [0m14:32:48  1 of 3 START sql view model bronze.raw_daily_customer_cdc ...................... [RUN]
[2023-12-30T14:32:48.712+0000] {subprocess.py:93} INFO - [0m14:32:48  2 of 3 START sql view model bronze.raw_daily_customer_drivers .................. [RUN]
[2023-12-30T14:32:48.712+0000] {subprocess.py:93} INFO - [0m14:32:48  3 of 3 START sql view model bronze.raw_daily_loan_transactions ................. [RUN]
[2023-12-30T14:32:48.877+0000] {subprocess.py:93} INFO - [0m14:32:48  2 of 3 OK created sql view model bronze.raw_daily_customer_drivers ............. [[32mCREATE VIEW[0m in 0.16s]
[2023-12-30T14:32:48.892+0000] {subprocess.py:93} INFO - [0m14:32:48  3 of 3 OK created sql view model bronze.raw_daily_loan_transactions ............ [[32mCREATE VIEW[0m in 0.18s]
[2023-12-30T14:32:48.895+0000] {subprocess.py:93} INFO - [0m14:32:48  1 of 3 OK created sql view model bronze.raw_daily_customer_cdc ................. [[32mCREATE VIEW[0m in 0.18s]
[2023-12-30T14:32:48.921+0000] {subprocess.py:93} INFO - [0m14:32:48
[2023-12-30T14:32:48.922+0000] {subprocess.py:93} INFO - [0m14:32:48  Finished running 3 view models in 0 hours 0 minutes and 0.30 seconds (0.30s).
[2023-12-30T14:32:49.000+0000] {subprocess.py:93} INFO - [0m14:32:48
[2023-12-30T14:32:49.001+0000] {subprocess.py:93} INFO - [0m14:32:49  [32mCompleted successfully[0m
[2023-12-30T14:32:49.002+0000] {subprocess.py:93} INFO - [0m14:32:49
[2023-12-30T14:32:49.002+0000] {subprocess.py:93} INFO - [0m14:32:49  Done. PASS=3 WARN=0 ERROR=0 SKIP=0 TOTAL=3
[2023-12-30T14:32:50.004+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-12-30T14:32:50.032+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=dbt_model, task_id=run_dbt_bronze_models, execution_date=20220910T000000, start_date=20231230T143246, end_date=20231230T143250
[2023-12-30T14:32:50.052+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-12-30T14:32:50.081+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-12-30T14:42:53.709+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dbt_model.run_dbt_bronze_models scheduled__2022-09-10T00:00:00+00:00 [queued]>
[2023-12-30T14:42:53.722+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dbt_model.run_dbt_bronze_models scheduled__2022-09-10T00:00:00+00:00 [queued]>
[2023-12-30T14:42:53.722+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-12-30T14:42:53.722+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 6
[2023-12-30T14:42:53.722+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-12-30T14:42:53.734+0000] {taskinstance.py:1383} INFO - Executing <Task(BashOperator): run_dbt_bronze_models> on 2022-09-10 00:00:00+00:00
[2023-12-30T14:42:53.738+0000] {standard_task_runner.py:55} INFO - Started process 17247 to run task
[2023-12-30T14:42:53.741+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'dbt_model', 'run_dbt_bronze_models', 'scheduled__2022-09-10T00:00:00+00:00', '--job-id', '147', '--raw', '--subdir', 'DAGS_FOLDER/run_dbt._models.py', '--cfg-path', '/tmp/tmpaaf62znl']
[2023-12-30T14:42:53.741+0000] {standard_task_runner.py:83} INFO - Job 147: Subtask run_dbt_bronze_models
[2023-12-30T14:42:53.784+0000] {task_command.py:376} INFO - Running <TaskInstance: dbt_model.run_dbt_bronze_models scheduled__2022-09-10T00:00:00+00:00 [running]> on host 683423f116e7
[2023-12-30T14:42:53.845+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=alonso_md
AIRFLOW_CTX_DAG_ID=dbt_model
AIRFLOW_CTX_TASK_ID=run_dbt_bronze_models
AIRFLOW_CTX_EXECUTION_DATE=2022-09-10T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-09-10T00:00:00+00:00
[2023-12-30T14:42:53.847+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-12-30T14:42:53.848+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', "    dbt run     --profiles-dir //opt/***/dags/../dbt/     --project-dir ///opt/***/dags/../dbt//dwh     --target prod_*** --vars '{ target_date: 2022-09-10 }' --select bronze"]
[2023-12-30T14:42:53.858+0000] {subprocess.py:86} INFO - Output:
[2023-12-30T14:42:55.865+0000] {subprocess.py:93} INFO - [0m14:42:55  Running with dbt=1.4.6
[2023-12-30T14:42:56.057+0000] {subprocess.py:93} INFO - [0m14:42:56  Found 8 models, 15 tests, 0 snapshots, 0 analyses, 703 macros, 0 operations, 0 seed files, 3 sources, 0 exposures, 0 metrics
[2023-12-30T14:42:56.059+0000] {subprocess.py:93} INFO - [0m14:42:56
[2023-12-30T14:42:56.131+0000] {subprocess.py:93} INFO - [0m14:42:56  Concurrency: 4 threads (target='prod_***')
[2023-12-30T14:42:56.131+0000] {subprocess.py:93} INFO - [0m14:42:56
[2023-12-30T14:42:56.139+0000] {subprocess.py:93} INFO - [0m14:42:56  1 of 3 START sql view model bronze.raw_daily_customer_cdc ...................... [RUN]
[2023-12-30T14:42:56.140+0000] {subprocess.py:93} INFO - [0m14:42:56  2 of 3 START sql view model bronze.raw_daily_customer_drivers .................. [RUN]
[2023-12-30T14:42:56.140+0000] {subprocess.py:93} INFO - [0m14:42:56  3 of 3 START sql view model bronze.raw_daily_loan_transactions ................. [RUN]
[2023-12-30T14:42:56.283+0000] {subprocess.py:93} INFO - [0m14:42:56  1 of 3 OK created sql view model bronze.raw_daily_customer_cdc ................. [[32mCREATE VIEW[0m in 0.14s]
[2023-12-30T14:42:56.287+0000] {subprocess.py:93} INFO - [0m14:42:56  3 of 3 OK created sql view model bronze.raw_daily_loan_transactions ............ [[32mCREATE VIEW[0m in 0.14s]
[2023-12-30T14:42:56.288+0000] {subprocess.py:93} INFO - [0m14:42:56  2 of 3 OK created sql view model bronze.raw_daily_customer_drivers ............. [[32mCREATE VIEW[0m in 0.15s]
[2023-12-30T14:42:56.352+0000] {subprocess.py:93} INFO - [0m14:42:56
[2023-12-30T14:42:56.352+0000] {subprocess.py:93} INFO - [0m14:42:56  Finished running 3 view models in 0 hours 0 minutes and 0.29 seconds (0.29s).
[2023-12-30T14:42:56.398+0000] {subprocess.py:93} INFO - [0m14:42:56
[2023-12-30T14:42:56.399+0000] {subprocess.py:93} INFO - [0m14:42:56  [32mCompleted successfully[0m
[2023-12-30T14:42:56.399+0000] {subprocess.py:93} INFO - [0m14:42:56
[2023-12-30T14:42:56.399+0000] {subprocess.py:93} INFO - [0m14:42:56  Done. PASS=3 WARN=0 ERROR=0 SKIP=0 TOTAL=3
[2023-12-30T14:42:57.632+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-12-30T14:42:57.674+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=dbt_model, task_id=run_dbt_bronze_models, execution_date=20220910T000000, start_date=20231230T144253, end_date=20231230T144257
[2023-12-30T14:42:57.729+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-12-30T14:42:57.785+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-12-30T14:52:50.289+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dbt_model.run_dbt_bronze_models scheduled__2022-09-10T00:00:00+00:00 [queued]>
[2023-12-30T14:52:50.298+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dbt_model.run_dbt_bronze_models scheduled__2022-09-10T00:00:00+00:00 [queued]>
[2023-12-30T14:52:50.299+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-12-30T14:52:50.299+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 6
[2023-12-30T14:52:50.299+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-12-30T14:52:50.311+0000] {taskinstance.py:1383} INFO - Executing <Task(BashOperator): run_dbt_bronze_models> on 2022-09-10 00:00:00+00:00
[2023-12-30T14:52:50.316+0000] {standard_task_runner.py:55} INFO - Started process 17770 to run task
[2023-12-30T14:52:50.319+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'dbt_model', 'run_dbt_bronze_models', 'scheduled__2022-09-10T00:00:00+00:00', '--job-id', '149', '--raw', '--subdir', 'DAGS_FOLDER/run_dbt._models.py', '--cfg-path', '/tmp/tmpw538t574']
[2023-12-30T14:52:50.319+0000] {standard_task_runner.py:83} INFO - Job 149: Subtask run_dbt_bronze_models
[2023-12-30T14:52:50.380+0000] {task_command.py:376} INFO - Running <TaskInstance: dbt_model.run_dbt_bronze_models scheduled__2022-09-10T00:00:00+00:00 [running]> on host 683423f116e7
[2023-12-30T14:52:50.445+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=alonso_md
AIRFLOW_CTX_DAG_ID=dbt_model
AIRFLOW_CTX_TASK_ID=run_dbt_bronze_models
AIRFLOW_CTX_EXECUTION_DATE=2022-09-10T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-09-10T00:00:00+00:00
[2023-12-30T14:52:50.446+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-12-30T14:52:50.446+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', "    dbt run     --profiles-dir //opt/***/dags/../dbt/     --project-dir ///opt/***/dags/../dbt//dwh     --target prod_*** --vars '{ target_date: 2022-09-10 }' --select bronze"]
[2023-12-30T14:52:50.455+0000] {subprocess.py:86} INFO - Output:
[2023-12-30T14:52:52.382+0000] {subprocess.py:93} INFO - [0m14:52:52  Running with dbt=1.4.6
[2023-12-30T14:52:52.485+0000] {subprocess.py:93} INFO - [0m14:52:52  Found 8 models, 15 tests, 0 snapshots, 0 analyses, 703 macros, 0 operations, 0 seed files, 3 sources, 0 exposures, 0 metrics
[2023-12-30T14:52:52.487+0000] {subprocess.py:93} INFO - [0m14:52:52
[2023-12-30T14:52:52.570+0000] {subprocess.py:93} INFO - [0m14:52:52  Concurrency: 4 threads (target='prod_***')
[2023-12-30T14:52:52.570+0000] {subprocess.py:93} INFO - [0m14:52:52
[2023-12-30T14:52:52.576+0000] {subprocess.py:93} INFO - [0m14:52:52  1 of 3 START sql view model bronze.raw_daily_customer_cdc ...................... [RUN]
[2023-12-30T14:52:52.581+0000] {subprocess.py:93} INFO - [0m14:52:52  2 of 3 START sql view model bronze.raw_daily_customer_drivers .................. [RUN]
[2023-12-30T14:52:52.582+0000] {subprocess.py:93} INFO - [0m14:52:52  3 of 3 START sql view model bronze.raw_daily_loan_transactions ................. [RUN]
[2023-12-30T14:52:52.657+0000] {subprocess.py:93} INFO - [0m14:52:52  1 of 3 ERROR creating sql view model bronze.raw_daily_customer_cdc ............. [[31mERROR[0m in 0.08s]
[2023-12-30T14:52:52.659+0000] {subprocess.py:93} INFO - [0m14:52:52  3 of 3 ERROR creating sql view model bronze.raw_daily_loan_transactions ........ [[31mERROR[0m in 0.07s]
[2023-12-30T14:52:52.660+0000] {subprocess.py:93} INFO - [0m14:52:52  2 of 3 ERROR creating sql view model bronze.raw_daily_customer_drivers ......... [[31mERROR[0m in 0.07s]
[2023-12-30T14:52:52.681+0000] {subprocess.py:93} INFO - [0m14:52:52
[2023-12-30T14:52:52.682+0000] {subprocess.py:93} INFO - [0m14:52:52  Finished running 3 view models in 0 hours 0 minutes and 0.19 seconds (0.19s).
[2023-12-30T14:52:52.692+0000] {subprocess.py:93} INFO - [0m14:52:52
[2023-12-30T14:52:52.693+0000] {subprocess.py:93} INFO - [0m14:52:52  [31mCompleted with 3 errors and 0 warnings:[0m
[2023-12-30T14:52:52.693+0000] {subprocess.py:93} INFO - [0m14:52:52
[2023-12-30T14:52:52.693+0000] {subprocess.py:93} INFO - [0m14:52:52  [33mDatabase Error in model raw_daily_customer_cdc (models/bronze/raw_daily_customer_cdc.sql)[0m
[2023-12-30T14:52:52.693+0000] {subprocess.py:93} INFO - [0m14:52:52    relation "airbyte.daily_customer_information_cdc" does not exist
[2023-12-30T14:52:52.694+0000] {subprocess.py:93} INFO - [0m14:52:52    LINE 9:     FROM "dwh"."airbyte"."daily_customer_information_cdc"
[2023-12-30T14:52:52.694+0000] {subprocess.py:93} INFO - [0m14:52:52                     ^
[2023-12-30T14:52:52.694+0000] {subprocess.py:93} INFO - [0m14:52:52    compiled Code at target/run/dwh/models/bronze/raw_daily_customer_cdc.sql
[2023-12-30T14:52:52.694+0000] {subprocess.py:93} INFO - [0m14:52:52
[2023-12-30T14:52:52.695+0000] {subprocess.py:93} INFO - [0m14:52:52  [33mDatabase Error in model raw_daily_loan_transactions (models/bronze/raw_daily_loan_transactions.sql)[0m
[2023-12-30T14:52:52.695+0000] {subprocess.py:93} INFO - [0m14:52:52    relation "airbyte.daily_loan_transactions" does not exist
[2023-12-30T14:52:52.695+0000] {subprocess.py:93} INFO - [0m14:52:52    LINE 9:     FROM "dwh"."airbyte"."daily_loan_transactions"
[2023-12-30T14:52:52.695+0000] {subprocess.py:93} INFO - [0m14:52:52                     ^
[2023-12-30T14:52:52.696+0000] {subprocess.py:93} INFO - [0m14:52:52    compiled Code at target/run/dwh/models/bronze/raw_daily_loan_transactions.sql
[2023-12-30T14:52:52.696+0000] {subprocess.py:93} INFO - [0m14:52:52
[2023-12-30T14:52:52.696+0000] {subprocess.py:93} INFO - [0m14:52:52  [33mDatabase Error in model raw_daily_customer_drivers (models/bronze/raw_daily_customer_drivers.sql)[0m
[2023-12-30T14:52:52.697+0000] {subprocess.py:93} INFO - [0m14:52:52    relation "airbyte.daily_customer_drivers" does not exist
[2023-12-30T14:52:52.697+0000] {subprocess.py:93} INFO - [0m14:52:52    LINE 9:     FROM "dwh"."airbyte"."daily_customer_drivers"
[2023-12-30T14:52:52.697+0000] {subprocess.py:93} INFO - [0m14:52:52                     ^
[2023-12-30T14:52:52.697+0000] {subprocess.py:93} INFO - [0m14:52:52    compiled Code at target/run/dwh/models/bronze/raw_daily_customer_drivers.sql
[2023-12-30T14:52:52.698+0000] {subprocess.py:93} INFO - [0m14:52:52
[2023-12-30T14:52:52.698+0000] {subprocess.py:93} INFO - [0m14:52:52  Done. PASS=0 WARN=0 ERROR=3 SKIP=0 TOTAL=3
[2023-12-30T14:52:53.695+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2023-12-30T14:52:53.706+0000] {taskinstance.py:1851} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 197, in execute
    f'Bash command failed. The command returned a non-zero exit code {result.exit_code}.'
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2023-12-30T14:52:53.719+0000] {taskinstance.py:1406} INFO - Marking task as UP_FOR_RETRY. dag_id=dbt_model, task_id=run_dbt_bronze_models, execution_date=20220910T000000, start_date=20231230T145250, end_date=20231230T145253
[2023-12-30T14:52:53.735+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 149 for task run_dbt_bronze_models (Bash command failed. The command returned a non-zero exit code 1.; 17770)
[2023-12-30T14:52:53.743+0000] {local_task_job.py:164} INFO - Task exited with return code 1
[2023-12-30T14:52:53.773+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-12-30T14:55:23.004+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dbt_model.run_dbt_bronze_models scheduled__2022-09-10T00:00:00+00:00 [queued]>
[2023-12-30T14:55:23.012+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dbt_model.run_dbt_bronze_models scheduled__2022-09-10T00:00:00+00:00 [queued]>
[2023-12-30T14:55:23.012+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-12-30T14:55:23.012+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 6
[2023-12-30T14:55:23.012+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-12-30T14:55:23.024+0000] {taskinstance.py:1383} INFO - Executing <Task(BashOperator): run_dbt_bronze_models> on 2022-09-10 00:00:00+00:00
[2023-12-30T14:55:23.029+0000] {standard_task_runner.py:55} INFO - Started process 17915 to run task
[2023-12-30T14:55:23.033+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'dbt_model', 'run_dbt_bronze_models', 'scheduled__2022-09-10T00:00:00+00:00', '--job-id', '150', '--raw', '--subdir', 'DAGS_FOLDER/run_dbt._models.py', '--cfg-path', '/tmp/tmph2kh8kxg']
[2023-12-30T14:55:23.033+0000] {standard_task_runner.py:83} INFO - Job 150: Subtask run_dbt_bronze_models
[2023-12-30T14:55:23.083+0000] {task_command.py:376} INFO - Running <TaskInstance: dbt_model.run_dbt_bronze_models scheduled__2022-09-10T00:00:00+00:00 [running]> on host 683423f116e7
[2023-12-30T14:55:23.138+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=alonso_md
AIRFLOW_CTX_DAG_ID=dbt_model
AIRFLOW_CTX_TASK_ID=run_dbt_bronze_models
AIRFLOW_CTX_EXECUTION_DATE=2022-09-10T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-09-10T00:00:00+00:00
[2023-12-30T14:55:23.139+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-12-30T14:55:23.139+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', "    dbt run     --profiles-dir //opt/***/dags/../dbt/     --project-dir ///opt/***/dags/../dbt//dwh     --target prod_*** --vars '{ target_date: 2022-09-10 }' --select bronze"]
[2023-12-30T14:55:23.150+0000] {subprocess.py:86} INFO - Output:
[2023-12-30T14:55:24.974+0000] {subprocess.py:93} INFO - [0m14:55:24  Running with dbt=1.4.6
[2023-12-30T14:55:25.075+0000] {subprocess.py:93} INFO - [0m14:55:25  Found 8 models, 15 tests, 0 snapshots, 0 analyses, 703 macros, 0 operations, 0 seed files, 3 sources, 0 exposures, 0 metrics
[2023-12-30T14:55:25.077+0000] {subprocess.py:93} INFO - [0m14:55:25
[2023-12-30T14:55:25.149+0000] {subprocess.py:93} INFO - [0m14:55:25  Concurrency: 4 threads (target='prod_***')
[2023-12-30T14:55:25.149+0000] {subprocess.py:93} INFO - [0m14:55:25
[2023-12-30T14:55:25.156+0000] {subprocess.py:93} INFO - [0m14:55:25  1 of 3 START sql view model bronze.raw_daily_customer_cdc ...................... [RUN]
[2023-12-30T14:55:25.157+0000] {subprocess.py:93} INFO - [0m14:55:25  2 of 3 START sql view model bronze.raw_daily_customer_drivers .................. [RUN]
[2023-12-30T14:55:25.158+0000] {subprocess.py:93} INFO - [0m14:55:25  3 of 3 START sql view model bronze.raw_daily_loan_transactions ................. [RUN]
[2023-12-30T14:55:25.290+0000] {subprocess.py:93} INFO - [0m14:55:25  1 of 3 OK created sql view model bronze.raw_daily_customer_cdc ................. [[32mCREATE VIEW[0m in 0.13s]
[2023-12-30T14:55:25.293+0000] {subprocess.py:93} INFO - [0m14:55:25  2 of 3 OK created sql view model bronze.raw_daily_customer_drivers ............. [[32mCREATE VIEW[0m in 0.13s]
[2023-12-30T14:55:25.294+0000] {subprocess.py:93} INFO - [0m14:55:25  3 of 3 OK created sql view model bronze.raw_daily_loan_transactions ............ [[32mCREATE VIEW[0m in 0.13s]
[2023-12-30T14:55:25.374+0000] {subprocess.py:93} INFO - [0m14:55:25
[2023-12-30T14:55:25.374+0000] {subprocess.py:93} INFO - [0m14:55:25  Finished running 3 view models in 0 hours 0 minutes and 0.30 seconds (0.30s).
[2023-12-30T14:55:25.421+0000] {subprocess.py:93} INFO - [0m14:55:25
[2023-12-30T14:55:25.422+0000] {subprocess.py:93} INFO - [0m14:55:25  [32mCompleted successfully[0m
[2023-12-30T14:55:25.422+0000] {subprocess.py:93} INFO - [0m14:55:25
[2023-12-30T14:55:25.422+0000] {subprocess.py:93} INFO - [0m14:55:25  Done. PASS=3 WARN=0 ERROR=0 SKIP=0 TOTAL=3
[2023-12-30T14:55:26.358+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-12-30T14:55:26.385+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=dbt_model, task_id=run_dbt_bronze_models, execution_date=20220910T000000, start_date=20231230T145523, end_date=20231230T145526
[2023-12-30T14:55:26.417+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-12-30T14:55:26.452+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
